README.md

```diff
@@ -1,28 +1,103 @@
 # StableHLO
 
-StableHLO is an operation set that expresses ML computations. It has been
-originally bootstrapped from
-[the MHLO dialect](https://github.com/tensorflow/mlir-hlo#meta-hlo-dialect-mhlo)
-and enhances it with additional functionality, including serialization and
-versioning.
+StableHLO is an operation set for high-level operations (HLO) in machine
```

Comment:
We don't mention MLIR but then we require MLIR in the build steps

Reply:
Okay, I added a mention for MLIR in the third paragraph.